### ISSUE TITLE
Use VF component instead of object-specific Pages

### ISSUE THREAD

**Issue by tet3** [bigthinks/weathervane#10]
Could probably enable extension to other objects, except the picklist on Indicator__mdt might be a bit of a problem. Maybe a "Custom" PL value and an additional text field.

**Comment by cdcarter** [comment]
yes, ok, so when the package came out mdt relations to entity definition were still in preview and couldn't be packaged so we had to use the picklist instead of a relationship. those are GA now, so we could certainly transition the package to use that. (preferred over a text field. i have no qualms about making a breaking change to the initial version :) )

resolving that would make this a very good idea.
